<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1389</width>
    <height>1066</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1260</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CAN Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1191</width>
      <height>761</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>1171</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>曲线分析</string>
     </property>
     <widget class="QPushButton" name="pushButton_Clear">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>80</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_StartTimer">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>40</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Clear_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>-50</y>
        <width>241</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
     <widget class="QChartView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1061</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>10</y>
       <width>241</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>设备总览</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>电机厂家：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>电机型号：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>钛虎</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>零差云控</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Elmo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>立迈胜</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>171</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string>钛虎电机文档</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_11">
         <property name="text">
          <string>零差电机文档</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_12">
         <property name="text">
          <string>Elmo电机文档</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_13">
         <property name="text">
          <string>立迈胜电机文档</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_14">
         <property name="text">
          <string>项目运用电机型号汇总</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>接口板信息</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>CAN通道数：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>硬件版本号:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>接口库版本号：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>中断号:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>321</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>数据发送</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>ID(0x):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sendIDEdit">
          <property name="text">
           <string>00000000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_4">
          <item>
           <property name="text">
            <string>通道0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>通道1</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox_6">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>标准帧</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>扩展帧</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_5">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>数据帧</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>远程帧</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>数据(0x,以空格隔开)：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sendDataEdit">
          <property name="text">
           <string>00 11 22 33 44 55 66 77</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>10</y>
       <width>211</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>运动控制</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>轮廓力矩模式</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>轮廓速度模式</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_8">
        <property name="text">
         <string>失能</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>轮廓位置模式</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_velocityctl_forward">
        <property name="text">
         <string>速度模式正转</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_motor_disable">
        <property name="text">
         <string>失能</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>设备类型：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBox_3">
          <item>
           <property name="text">
            <string>10Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>40Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>80Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>125Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>200Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>250Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>400Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>666Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>800Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1000Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>33.33Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>66.66Kbps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>83.33Kbps</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>波特率：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>复位</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButton_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>启动CAN</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>初始化CAN</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>USBCAN-2A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>USBCAN-2C</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CANalyst-II</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>设备索引：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>打开设备</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>1171</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>数据接收</string>
     </property>
     <widget class="QPushButton" name="cleanListBtn">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>70</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1061</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>30</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>实时显示</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
